e2e/reporters/reporter.js

```diff
@@ -48,14 +48,12 @@ class SnapshotAwareReporter {
     const filename = path.basename(testPath);
     const name = ancestorTitles.join(' > ');
     const statBase = {name, filename, status, title, time: duration};
-
     if (status === 'passed') return statBase;
 
-    const snapshotParts = [filename, ...ancestorTitles, title, '1-snap', 'diff'];
-    const snapshotName = snapshotParts.map(sanitize).join('-') + '.png';
+    const snapshotParts = [filename, ...ancestorTitles, title, 'diff'];
```

Comment:
That cool for printing nut just one more highlight: 
 we don't use `ancestorTitles` to build the name, as it was not used in serializer
 we need exactly the same way to get file name